### PR TITLE
Validate VLC path if pasted from clipboard

### PR DESCRIPTION
**Pull request by ivandrofly**
Portable VLC

### CODE REVIEW

File: src/Forms/Settings.cs [hunk 1]
```diff
@@ -2553,5 +2558,18 @@ private void linkLabelOpenDictionaryFolder_LinkClicked(object sender, LinkLabelL
             System.Diagnostics.Process.Start(dictionaryFolder);
         }
 
+        private void textBoxVlcPath_MouseLeave(object sender, EventArgs e)
+        {
+            try
+            {
+                var path = textBoxVlcPath.Text.Trim('\"');
+                if (path.Length > 3 && Path.IsPathRooted(path) && Path.GetFileName(path).Equals("vlc.exe", StringComparison.OrdinalIgnoreCase) && File.Exists(path))
+                {
```
Comment (xylographe) [inline src/Forms/Settings.cs]: The .exe extension is an MS Windows peculiarity. This test might fail needlessly on a different OS.

Comment (ivandrofly) [inline src/Forms/Settings.cs]: So why open file dialogue is filtering only ```*.vlc.exe``` :confused:

Comment (xylographe) [inline src/Forms/Settings.cs]: I would assume, that might be the best guide for clueless users?
It wouldn't stop you, though, from entering something completely different. :smile:

Comment (ivandrofly) [inline src/Forms/Settings.cs]: @niksedk

Comment (niksedk) [inline src/Forms/Settings.cs]: I guess the portable stuff is only relevant for windows users, right?
Also, I've not succeeded in making VLC work as video player on linux (or osx - don't have a mac).

**Pull request merged**